simplify README.md and run_local.sh (#1096)
linera-io/linera-protocol: README.md

## Repository Structure

> The Linera protocol repository is broken down into the following main crates and subdirectories: (from low-level to high-level in the dependency graph)

Reviewer: ```suggestion
The Linera protocol repository comprises the following main crates and subdirectories: (from low-level to high-level in the dependency graph)
```

(Or "consists of"?)

10. [`examples`](./examples) Examples of Linera applications written in Rust.

Additionally,

> - `linera-service-graphql-client` is a Rust GraphQL client for the system components of the `linera-service`

Reviewer: ```suggestion
- `linera-service-graphql-client` is a Rust GraphQL client for the system components of the `linera-service`.
```

- `linera-explorer` contains an experimental block explorer.
- `linera-indexer` is an experimental block indexer.
- `linera-witty` is an experimental alternative to `wit-bindgen` for rust hosts.

## Quickstart

See our [developer manual](https://linera.dev) for a full introduction to the Linera SDK.

The following script creates a local test network and runs a simple transfers between two user chains.

It can be run with `cargo test readme`.

```bash
if [ $# -eq 1 ]
then
    STORAGE="$1"
else

>     STORAGE="ROCKSDB"

Reviewer: I'd rather not support databases other than RocksDB in the README example, and explain those in the manual instead. It would make the README a lot more readable.
